packages/cli/src/cmds/archive/indexAppMaps.ts

```diff
@@ -15,7 +15,7 @@ export async function indexAppMaps(
   options.fileCountFn = counter.setCount();
   options.errorFn = reportAppMapProcessingError('Index');
   await processFiles(
-    '**/*.appmap.json',
+    '.appmap.json',
```

Comment:
You'll see a bunch of this in the PR. We are using an old version of Glob. It's given me fits when the `appmap_dir` is `.`, as it is in some tests and may possibly be in the real world? The only feature of Glob we were using is recursive finding by file name or file extension, so I have just written that and refactored Glob out.